python_modules/dagster-test/dagster_test/toys/asset_checks.py

```diff
@@ -412,11 +410,6 @@ def downstream_asset():
     return 1
 
 
-just_checks_job = define_asset_job(
```

Comment:
I don't get my toy?

Haha yeah now that this is selecting checks in assets as well, could we change this to `AssetSelection.asset_checks_for_assets(checked_asset)`?